Java/cp/Longest_Pallindromicsubstring.java

```diff
@@ -0,0 +1,50 @@
+// Java solution for longest palindrome in a substring
+import java.util.*;
+ 
+class PallindromicSubstr{
+ 
+// Function to print longest subString
+static void printSubStr(String str, int low, int high)
+{
+    for (int i = low; i <= high; ++i)
+        System.out.print(str.charAt(i));
+}
+
+static int longestPallindromeSubstr(String str)
+{
+    // get length of input String
```

Comment:
Isn't the comment kinda redundant? 

![image](https://user-images.githubusercontent.com/47351025/107845725-ddcba480-6e03-11eb-8497-af9e0aae9278.png)